src/views/ResourceInfo.js

```diff
@@ -0,0 +1,200 @@
+    organizations: ["Montreal AI Ethics Institute"]
+  };
+
+function FileTable(props) {
+  const columns = [
+    {
+      title: 'Name',
+      dataIndex: 'name',
+      key: 'name',
+    },
+    {
+      title: 'Type',
+      dataIndex: 'type',
+      key: 'type',
+    },
+    {
+      title: 'Link',
+      dataIndex: 'link',
+      key: 'link',
+      render: text => <a href={text}>{text}</a>
+    },
+  ]
+  return (
+    <div>
+      <h1 style={{ padding:'10px',fontSize: '2em', fontWeight: 'bold'}}>Files</h1>
+      <Table columns={columns} dataSource={props.data} />
+    </div>
+  )
+}
+
+function SideBar(props){
+  return(
+  <Affix offsetTop={60}>
+    <Sider width={250}>
+      <Menu
+        mode="inline"
+        theme="light"
+        defaultOpenKeys={['overview']}
+        style={{ height: '100%', borderRight: 0 }}
+      >
+        <Menu.Item 
+          key="overview" 
+          icon={<FileDoneOutlined/>}
+          style={{ marginTop: '30px' }}
+          onClick={() => {
+            props.topRef.current.scrollIntoView();
+          }}>
+            Overview
+        </Menu.Item>
+
+        <Menu.Item 
+          key="details"  
+          icon={<SearchOutlined/>}                     
+          onClick={() => {
+            props.detailRef.current.scrollIntoView();
+          }} >
+            Details 
+        </Menu.Item>
+        <Menu.Item 
+          key="files"
+          icon={<FolderOpenOutlined/>}
+          onClick={() => {
+            props.fileRef.current.scrollIntoView();
+          }} >
+            Files 
+        </Menu.Item>
+      </Menu>
+    </Sider>
+  </Affix> );
+}
+
+
+export default function ShowExample() {
+  return (
+    <ResourceInfo resource = {resourceData}/>
+  );
+}
+
+function ResourceInfo(props) {
+  let topRef = useRef(null);
+  let fileRef = useRef(null);
+  let detailRef = useRef(null);
+  let children= [];
+  for (let i = 0; i < props.resource.topics.length; i++) {
+    children.push(<Tag color='blue'> {props.resource.topics[i]}</Tag>)
+  }
+
+  return (
+    <Layout style={{ height: `${window.innerHeight}px`, overflow: 'hidden', }}>
+      <FormHeader/>
+      <Layout>
+        <SideBar topRef={topRef} fileRef={fileRef} detailRef={detailRef}/>
+        <Content>
+          <div ref = {topRef}>
+            <PageHeader
+              title={props.resource.name}
+              onBack={() => window.history.back()}
+              className="site-page-header"
+              subTitle={props.resource.organizations.join(", ")}
+              tags={children}
+              avatar={{ src: props.resource.avatarIcon}}
+              extra={[
+                <Button 
+                  icon={  <EditOutlined />}
+                  key="3" 
+                  shape="round">
+                    Edit Resource
+                </Button>,
+              ]}
+            >
+              {props.resource.desc}
+            </PageHeader>,
+          </div>
+          <div ref={detailRef}>
```

Comment:
Can we add more padding to the left and right of this component. Right now it's flush with the sides but there should be a small gap.